src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+@import 'header';
+@import 'variables';
+@import 'product_card';
+@import 'stars_rating';
+@import 'cards_container';
+
+html {
+  font-family: Roboto, sans-serif, 'Times New Roman';
+
+  --active-link-color: #00acdc;
```

Comment:
The variable `--active-link-color` should be declared in the `variables.scss` file. It's a good practice to keep all your variables in one place for better maintainability.